apps/frontend/src/app/5_pages/AavePage/components/BorrowAssetsList/BorrowAssetsList.tsx

```diff
@@ -34,7 +53,16 @@ export const BorrowAssetsList: FC<BorrowAssetsListProps> = ({
     ),
     [],
   );
-  const mobileRenderer = useCallback(p => <BorrowAssetDetails pool={p} />, []);
+
+  const mobileRenderer = useCallback(
+    p => (
+      <BorrowAssetDetails
+        onBorrowClick={() => onBorrowClick(p.asset)}
```

Comment:
onBorrowClick useCallback is useless because a new function is created on every render here

Reply:
I can change prototype of `BorrowAssetDetails` to receive (asset) => void  but it would be moving this elsewhere. Idk, any ideas? the whole useCallback for a one liner seems a bit too much for me haha